lua/org-roam/core/utils/io.lua

```diff
@@ -56,6 +56,11 @@ end
 ---@param data string|string[]
 ---@param cb fun(err:string|nil)
 function M.write_file(path, data, cb)
+    -- creat file directory with 0o644 (rw-r--r--) if not exist
+    dir = vim.fs.dirname(path)
```

Comment:
This should be `local dir = vim.fs.dirname(path)`, otherwise it creates a global variable.

Reply:
My bad, now fixed, thank you.